proofs/sectorsize.go

```diff
@@ -0,0 +1,19 @@
+package proofs
+
+// #cgo LDFLAGS: -L${SRCDIR}/../proofs/lib -lfilecoin_proofs
+// #cgo pkg-config: ${SRCDIR}/../proofs/lib/pkgconfig/libfilecoin_proofs.pc
+// #include "../proofs/include/libfilecoin_proofs.h"
+import "C"
+import (
+	"unsafe"
+)
+
+func SectorSize(ssType Mode) uint64 {
```

Comment:
Is this only used in test code? Is there future utility in it for production code? If not, it would be nice if this could be moved into a test package. I'm not sure how hard that would be.

CC @laser

Reply:
This is definitely a useful production artifact of this PR that will put into use to resolve #2034

Reply:
> Is this only used in test code?

No, this will be used by storage clients to validate that the piece they'd like to make a deal for will fit into the sector size offered by the miner which owns the ask.

It will also be used by the storage miner to validate that the bytes that they were given by a storage client will fit into the sector that they've offered.

It will also be used in test code to create pieces which pack in a certain way (i.e. trigger sealing by filling up a freshly-provisioned yet-to-be-sealed sector).